Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1250,6 +1250,10 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   }
 
   override func visit(_ node: MacroExpansionExprSyntax) -> SyntaxVisitorContinueKind {
+    before(
```

Comment:
Before this, we need to pick up the same logic we're doing here: https://github.com/apple/swift-format/blob/8793d965ecd830bae26d86c6cddca607a3a7fa93/Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift#L981-L985

And then pass that value into `forcesBreakBeforeRightDelimiter` below.

Reply:
Thank you for the feedback! I have pushed an update addressing the suggestion and adding another test case similar to the [corresponding one](https://github.com/apple/swift-format/blob/8793d965ecd830bae26d86c6cddca607a3a7fa93/Tests/SwiftFormatPrettyPrintTests/FunctionCallTests.swift#L276) in `FunctionCallTests.swift`.